nautobot/extras/management/__init__.py

```diff
@@ -243,10 +243,12 @@ def _create_custom_role_or_status_instances(
 
     # Prep the app and get the model dynamically
     try:
-        Metadata_Model = apps.get_model(f"extras.{metadata_model}")
+        MetadatModel = apps.get_model(f"extras.{metadatamodel}")
```

Comment:
`MetadataModel` not `MetadatModel`.

I didn't mean that you also needed to change `metadata_model` to `metadatamodel` but it's fine either way.